SU2_CFD/include/SU2_CFD.hpp

```diff
@@ -39,7 +39,7 @@
 #pragma once
 
 #include "../../Common/include/mpi_structure.hpp"
-#include <CLI11.hpp>
+#include "../../externals/CLI11/CLI11.hpp"
```

Comment:
Is this what we do for other externals or is adding to the include path more appropriate?